payments-core/src/test/java/com/stripe/android/view/AddPaymentMethodViewModelTest.kt

```diff
@@ -29,6 +30,54 @@ class AddPaymentMethodViewModelTest {
 
     private val customerSession: CustomerSession = mock()
 
+    @Test
+    fun `on init without being launched from another product, should regenerate session id`() {
```

Comment:
Looks like we're testing all the side effects, but not the thing that causes the side effect.

Can we also test that launching from another product launches with the correct args?